aiven/client/cli.py

```diff
@@ -3863,6 +3865,11 @@ def service__create(self) -> None:
                         "source_service": self.args.read_replica_for,
                     }
                 )
+        elif self.args.recovery_target_time and self.args.service_to_fork_from:
+            user_config["service_to_fork_from"] = self.args.service_to_fork_from
+            user_config["recovery_target_time"] = self.args.recovery_target_time
+        elif self.args.service_to_fork_from:
+            user_config["service_to_fork_from"] = self.args.service_to_fork_from
```

Comment:
```suggestion
        elif self.args.service_to_fork_from:
            user_config["service_to_fork_from"] = self.args.service_to_fork_from
            if self.args.recovery_target_time:
                user_config["recovery_target_time"] = self.args.recovery_target_time
```

Reply:
Also I'm wondering where we validate `user_config`?

Reply:
I think that's the job for the backend. That makes it less likely for us to need to update this client if we add/change somehting there.

I personally would just put all values given on command line to the dict unconditionally. If some combination of them is invalid, then it's the APIs job to tell us.